internal/packager/cnab.go

```diff
@@ -162,7 +165,14 @@ func ToCNAB(app *types.App, invocationImageName string) (*bundle.Bundle, error)
 			},
 		},
 		Images: bundleImages,
-	}, nil
+	}
+
+	logrus.Debug("App converted to CNAB")
+	if js, err := json.Marshal(bndl); err == nil {
+		logrus.Debug(string(js))
```

Comment:
I think you can have multi line debug here. So you can merge the 2 calls to debug?